src/Microsoft.SqlTools.ResourceProvider/Microsoft.SqlTools.ResourceProvider.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <PackageId>SqlToolsResourceProviderService</PackageId>
+    <AssemblyName>SqlToolsResourceProviderService</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <Company>Microsoft</Company>
+    <Product>Sql Tools Service for Resource Provider services</Product>
+    <Description>Provides Resource Provider and control plane support.</Description>
+    <Copyright>� Microsoft Corporation. All rights reserved.</Copyright>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
+    <DebugType>portable</DebugType>
+	  <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
+    <RuntimeIdentifiers>win7-x64;win7-x86;ubuntu.14.04-x64;ubuntu.16.04-x64;centos.7-x64;rhel.7.2-x64;debian.8-x64;fedora.23-x64;opensuse.13.2-x64;osx.10.11-x64</RuntimeIdentifiers>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>TRACE;DEBUG;NETCOREAPP1_0;NETCOREAPP2_0</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.SqlTools.Hosting\Microsoft.SqlTools.Hosting.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Localization\sr.resx" />
+    <None Include="Localization\sr.strings" />
+  </ItemGroup>
+</Project>
```

Comment:
you need to add the project to codecoverage.bat if you want to see code coverage on the classes in this project

Reply:
Thanks, I didn't realize this. Added this. If we add a lot more DLLs in the future, we'll want to streamline this (e.g. having a different build configuration and setting these via that so we just change the property and it all works).